school_center.py

```diff
@@ -68,27 +71,53 @@ def sort_key(c):
     else: # if there are no centers within given  threshold, return one that is closest
         return [center_to_dict(nearest_center, nearest_distance)]
 
+
 def read_tsv(file_path: str) -> List[Dict[str, str]]:
     data = []
-    with open(file_path, 'r', newline='', encoding='utf-8') as file:
-        reader = csv.DictReader(file, delimiter='\t')
-        for row in reader:
-            data.append(dict(row))
+    try:
+        with open(file_path, 'r', newline='', encoding='utf-8') as file:
+            reader = csv.DictReader(file, delimiter='\t')
+            for row in reader:
+                data.append(dict(row))
+    except FileNotFoundError as e:
+        logging.error(f"Error: File '{file_path}' not found.")
```

Comment:
The message does not need to mention "Error" as the logline itself is an error, so
`logger.error(f"File '{file_path}' not found.")` would be enough, and the same for others.